<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="DESKTOP-181TTAT" failures="1" tests="7" name="Test_All_Modules_edge" time="59.164" errors="0" timestamp="2025-08-07T00:31:14 GMT-04:00">
  <testcase classname="testCases.Stack" name="open_stac_page" time="0.838"/>
  <testcase classname="testCases.Queue" name="open_queue_page" time="0.698"/>
  <testcase classname="testCases.Stack" name="stackOperations" time="1.387"/>
  <testcase classname="testCases.Queue" name="queueImplementation" time="1.169"/>
  <testcase classname="testCases.Stack" name="StackoperationsCodeEditor" time="2.421"/>
  <testcase classname="testCases.Queue" name="queueImplementationCodeEditor" time="1.979"/>
  <testcase classname="testCases.Stack" name="StackoperationsTryingEmptyEditor" time="2.339">
    <failure type="java.lang.AssertionError" message="Failing this test case to show the bug which is, no alert message comes up when clicking on run button without entering anhy code in it">
      <![CDATA[java.lang.AssertionError: Failing this test case to show the bug which is, no alert message comes up when clicking on run button without entering anhy code in it
at testCases.Stack.StackoperationsTryingEmptyEditor(Stack.java:72)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:317)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)
at java.base/java.lang.Thread.run(Thread.java:1583)
... Removed 16 stack frames]]>
    </failure>
  </testcase> <!-- StackoperationsTryingEmptyEditor -->
</testsuite> <!-- Test_All_Modules_edge -->
